### PR TITLE
start farmer node for gemini-3c

### PR DESCRIPTION
**Pull request by vedhavyas**
This PR starts a farmer for Gemini-3c

### CODE REVIEW

**Review by nazar-pc — COMMENTED** [review]
I'm really not sure we want to run farmer for Gemini 3c, it is devs and community who should run them

File: network-primitives/variables.tf [hunk 1]
```diff
@@ -13,6 +13,12 @@ variable "datadog_api_key" {
   type        = string
 }
 
+variable "piece_cache_size" {
+  description = "Datadog API Key"
```
Comment (nazar-pc) [inline network-primitives/variables.tf]: Copy-pasted comment

File: resources/gemini-3c/main.tf [hunk 2]
```diff
@@ -4,52 +4,53 @@ module "gemini-3c" {
...
     docker-org             = "subspace"
-    docker-tag             = "gemini-3c-2023-jan-18"
+    docker-tag             = "gemini-3c-2023-jan-27"
     reserved-only          = false
     plot-size              = "10G"
     reward-address         = var.farmer-reward-address
     force-block-production = false
     prune                  = false
   }
+  piece_cache_size     = "500KiB"
```
Comment (nazar-pc) [inline resources/gemini-3c/main.tf]: Why 500KiB? We want something like 10G at least instead. I was talking about 500k pieces, not bytes.

**Comment by vedhavyas** [comment]
> I'm really not sure we want to run farmer for Gemini 3c, it is devs and community who should run them

Fair but lets keep one running so that we don't let the chain stall.

**Review by nazar-pc — APPROVED** [review]

**Pull request merged**